Chat/Chat.cpp

```diff
@@ -0,0 +1,168 @@
+#include "Chat.h"
+#include "Network.h"
+#include <iostream>
+#include <thread>
+#include <chrono>
+
+#include <conio.h>
+
+using std::cout;
```

Comment:
i use it only for cout, cin and endl (most used). In all other cases I use std:: .